frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/components/SummaryModal.js

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect, useState } from "react";
+import { CardText, Modal } from "@egovernments/digit-ui-react-components";
+import { TextArea } from "@egovernments/digit-ui-components";
+import { useTranslation } from "react-i18next";
+
+const fieldStyle = { marginRight: 0 };
+
+const Heading = (props) => {
+  return (
+    <div style={{ width: "824px", height: "28px" }}>
+      <p
+        style={{
+          fontWeight: 700,
+          fontSize: "24px",
+          lineHeight: "28.13px",
+          color: "#0A0A0A",
+        }}
+      >
+        {props.label}
+      </p>
+    </div>
+  );
+};
+
+const Close = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#0A0A0A">
+    <path d="M0 0h24v24H0V0z" fill="none" />
+    <path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z" />
+  </svg>
+);
+
+const ForwardArrowIcon = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="currentColor" class="bi bi-arrow-right-short" viewBox="0 0 16 16">
+    <path
+      fill-rule="evenodd"
+      d="M4 8a.5.5 0 0 1 .5-.5h5.793L8.146 5.354a.5.5 0 1 1 .708-.708l3 3a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708-.708L10.293 8.5H4.5A.5.5 0 0 1 4 8"
+    />
+  </svg>
+);
+
+const AlertIcon = () => {
+  return (
+    <svg width="20" height="20" viewBox="0 0 24 24" fill="#0F3B8C" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
+        fill="#0F3B8C"
+      />
+      <path d="M11 10h2v6h-2zm0-3h2v2h-2z" fill="#0F3B8C" />
+    </svg>
+  );
+};
+
+const BackBtn = ({ text }) => {
+  return (
+    <div style={{ display: "flex", alignItems: "center", justifyContent: "center" }}>
+      <div>{text}</div>
+      <div style={{ width: "24px", height: "24px", marginLeft: "4px" }}>
+        <ForwardArrowIcon />
+      </div>
+    </div>
+  );
+};
+
+const CloseBtn = (props) => {
+  return (
+    <div onClick={props?.onClick} style={{ paddingTop: 10 }}>
+      <div className={"icon-bg-secondary"} style={{ backgroundColor: "#ffff", cursor: "pointer" }}>
+        {" "}
+        <Close />{" "}
+      </div>
+    </div>
+  );
+};
+
+const SummaryModal = ({ handleConfirmationModal, hearingId }) => {
+  const { t } = useTranslation();
+  const tenantId = window?.Digit.ULBService.getCurrentTenantId();
+  const [transcript, setTranscript] = useState("");
+
+  const { data: latestText, refetch } = Digit.Hooks.hearings.useGetHearings(
+    { hearing: { tenantId } },
+    { applicationNumber: "", cnrNumber: "", hearingId },
+    "dristi",
+    true
+  );
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        await refetch();
```

Comment:
why double fetch ? useGetHearings should already fetch the data, no need for useEffect and refetch